tests/test_vlan.rb

```diff
@@ -273,6 +273,7 @@ def test_vlan_add_remove_interface_valid
     count = 3
     interfaces.each do |name, interface|
       next unless interface.name.match(%r{ethernet[0-9/]+$}) && count > 0
+      interface_ethernet_default(%r{net(\d+\/\d+)}.match(name)[1])
```

Comment:
nit: you don't have to escape the slash if you're using %r, ie:   `%r{net(\d+/\d+)}`